index.html

```diff
@@ -55,5 +55,175 @@ <h1 class="headline__title">I’m Ahmed Glad to see you!</h1>
         </li>
       </ul>
     </section>
+
+    <section class="works">
+      <div class="works__single">
+        <img
+          class="works__single__img"
+          src="images/project2.png"
+          alt="project 1"
+        />
+
+        <p class="works__single__title">Multi-Post Stories</p>
+
+        <div class="works__single__details">
+          <p
```

Comment:
- [ ] [REQUIRED] Please ensure that this project title uses a header tag since header tags are used to separate headings and subheadings on a webpage and they improve the readability and SEO of a webpage 😊